ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js

```diff
@@ -20,6 +20,20 @@ const tankBusterOnParty = (data, matches) => {
   return data.party.inParty(data.target);
 };
 
+const getFacing = (combatant) => {
+  // Snap heading to closest card.
+  // N = 0, E = 1, S = 2, W = 3
+  return (2 - Math.round(combatant.Heading * 4 / Math.PI) / 2) % 4;
```

Comment:
I think truncating like so the problem?
Math.trunc(2 - Math.round(heading * 4 / Math.PI) / 2) % 4

Reply:
I am going to leave this issue in for now and hope the delaySeconds 1 resolves it. The mechanic has plenty of time to resolve. It should at least always call out the adjacent spots correctly now, but as those are relative you still need someone to give you a safe spot callout else you are dealing with a 1 in 4 odds.